.travis.yml

```diff
@@ -145,6 +153,7 @@ jobs:
 
       before_install:
       # See: https://github.com/containerd/cri/blob/master/docs/installation.md
+      - sudo systemctl stop docker
```

Comment:
I think we need to split up the build and run phase I assume that the docker installation and containerd are in conflict.